.gitignore

```diff
@@ -15,3 +15,4 @@ pytestdebug.log
 ubuntu-xenial-16.04-cloudimg-console.log
 .python-version
 coverage.xml
+.vscode/
```

Comment:
Might be nice to clean this up, if everyone is adding their IDE to the gitignore the file will grow humongous :)

Reply:
@wilmardo , how would you suggest we do that? A comment to introduce a section for IDE settings? Or just remove it alltogether?

Reply:
In my opinion just remove it altogether but the Ansible guys might have something to say about that ;)

Reply:
I just learned that there is a local gitignore that I can configure (`.git/info/exclude`). Feels more appropriate. I'll revert the commit.

Reply:
Done